test/script.py

```diff
@@ -16,14 +16,9 @@
 from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
 
 class bcolors:
-    HEADER = '\033[95m'
-    OKBLUE = '\033[94m'
     OKGREEN = '\033[92m'
-    WARNING = '\033[93m'
     FAIL = '\033[91m'
     ENDC = '\033[0m'
-    BOLD = '\033[1m'
-    UNDERLINE = '\033[4m'
```

Comment:
I think this should be moved out into a separate PR, it would be merged much faster in this case.